autogpt_platform/backend/backend/server/rest_api.py

```diff
@@ -67,10 +67,12 @@ def run_service(self):
 
         app.add_middleware(
             CORSMiddleware,
-            allow_origins=["*"],  # Allows all origins
+            allow_origins=[
+                "http://localhost:3000",
+                "http://127.0.0.1:3000",
+                "https://dev-builder.agpt.co",
```

Comment:
If we put `BACKEND_CORS_ALLOW_ORIGINS=http://localhost:3000` in `.env.template` and configure it as `https://dev-builder.agpt.co` in our deployment, we can get rid of these hardcoded values.